procrastinate/worker.py

```diff
@@ -36,6 +36,9 @@ def __init__(
     async def run(self) -> None:
         await self.job_store.listen_for_jobs(queues=self.queues)
 
+        self.logger.debug(
+            f"Log start one queues {self.queues}", extra={"queues": str(self.queues)}
```

Comment:
All log extras should have an unique "action" (at least, that's what has been done elsewhere)